README.md

```diff
@@ -87,6 +82,10 @@ KnowSheet> HTTP(2015)
 // KnowSheet Bricks HTTPServer at port 2015
 ```
 
+* Bricks maintains a single server per port. The first call to `HTTP` with a port number starts an HTTP server on that port and returns an instance of the server. The next calls with the same port reference the started server.
+* The calls to the server methods can be chained (each method returns the server instance).
+* Each server accepts connections since start till the process exits (there is no way to explicitly stop it).
```

Comment:
Nit: `(there is no way ...) ` sounds a bit too negative for US taste.

Perhaps, a milder version of "Therefore, there is neither a need nor a way to stop the server listening on certain port." could work?